addons/web/controllers/main.py

```diff
@@ -1671,9 +1671,10 @@ def get_fields(self, model, prefix='', parent_name= '',
                    parent_field=None, exclude=None):
 
         fields = self.fields_get(model)
+        rec_name = request.env[model]._rec_name
         if import_compat:
             if parent_field_type in ['many2one', 'many2many']:
-                fields = {'id': fields['id'], 'name': fields['name']}
+                fields = {k: v for k, v in fields.items() if k in ['id', rec_name]}
```

Comment:
```suggestion
                fields = {'id': fields['id'], 'name': fields[rec_name]}
```
Actually if the `_rec_name` is not present, we do want an error, we don't want a dictionary with only element

Reply:
Are you sure about that ? If _rec_name is not present, user cannot expand m2x fields in export screen, system will give error again and again when user click to these fields.

Reply:
@phongdao do you have valid cases where there is no _rec_name attribute? It should always be present, is by default `name` and uses a fallback to `id` if not present. If you have neither of these fields, you will probably have bigger issues than being able to expand m2x fields. Silently hiding an abnormal behaviour should be avoided.

Reply:
I have pushed new code as suggestion. Tks @mart-e